### PR TITLE
Expands translation support

### PR DESCRIPTION
**Pull request by wackerow**
## Description
Refactored the following for translation support

- [x] EthExchanges
- [x] FileContributors
- [x] ProductList
- [x] SectionNav
- [x] TranslationsInProgress
- [x] TutorialMetadata
- [x] languages.js
- [x] docs.js

## Related Issue #2006

### CODE REVIEW

File: src/components/EthExchanges.js [hunk 1]
```diff
@@ -439,11 +441,15 @@ const EthExchanges = () => {
         // Add state exceptions if Country is USA
         let description = null
         if (
-          state.selectedCountry.country === "United States of America (USA)"
+          state.selectedCountry.country ===
```
Comment (samajammin) [inline src/components/EthExchanges.js:444]: Note: this actually doesn't need translation as it'd not a user-facing string 😄

File: src/components/EthExchanges.js [hunk 2]
```diff
@@ -473,11 +479,15 @@ const EthExchanges = () => {
             // Add state exceptions if Country is USA
             let description = null
             if (
-              state.selectedCountry.country === "United States of America (USA)"
+              state.selectedCountry.country ===
```
Comment (samajammin) [inline src/components/EthExchanges.js:482]: Same here.

File: src/intl/en.json [hunk 3]
```diff
@@ -129,6 +129,29 @@
   "nav-use-aria-label": "Use Ethereum menu",
   "nav-learn-aria-label": "Learn menu",
   "on-this-page": "On this page",
+  "comp-tutorial-metadata-tip-author": "TIP AUTHOR",
```
Comment (samajammin) [inline src/intl/en.json:132]: We should probably format this text to uppercase vs. storing here as uppercase.

**Review by samajammin — APPROVED** [review]
AMAZING! Thanks for this & for the quick turnaround ❤️ 🏆

File: src/intl/en.json (hunk 3 above)
Comment (wackerow) [reply src/intl/en.json:132]: Yeah not a bad thought... I bet there are some others in there that are duplicated only to change the case that could potentially be styled to reduce redundancies. Let me know, I could post it as an issue if you want and take a look

**Pull request merged**